src/blog/2023/06/connect-to-hivemq-in-flowforge.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: MQTT and its Role in IoT and Industrial IoT
+title: Connect to an MQTT Broker - HiveMQ Cloud with Node-RED
```

Comment:
```suggestion
title: Connect Node-RED to HiveMQ Cloud an MQTT Broker
```

Reply:
Which will be a stronger search?  Connect Node-RED to HiveMQ, or Connect Node-RED to an MQTT Broker?